client/client-multi/client-c/src/kaa/utilities/kaa_log.h

```diff
@@ -29,6 +29,8 @@
 #include "../kaa_error.h"
 #include "../platform/defaults.h"
 
+#include <stdio.h>
```

Comment:
As I remember this header wasn't added intentionally, because of compatibility with Econais.
But at other hand, as I know now Kaa isn't going to support this platform in the future.